ydb/library/yql/public/udf/arrow/bit_util.h

```diff
@@ -47,6 +47,32 @@ inline void CompressSparseImpl(ui8* dst, const ui8* srcSparse, size_t len) {
     }
 }
 
+inline void DecompressToSparseBitmap(ui8* dstSparse, const ui8* src, size_t srcOffset, size_t len) {
+    if (srcOffset != 0) {
+        for (ui8 i = srcOffset; i < 8 - srcOffset; i++) {
```

Comment:
i < 8 ?

Reply:
Reimplemented this routine. `srcOffset` is processed [another way](https://github.com/ydb-platform/ydb/pull/3400/files#diff-b8e5dc41387f49b5eb79979cfe9dc8c2cfccf15491225b4801cbdb37fcb778d8R19-R28).